phantoms/brain/sim_brain_phantom.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import json
+import numpy as np
+import nibabel as nib
+from scipy.ndimage import zoom
+
+regime = 'diffusive'#'ballistic' # 
```

Comment:
Maybe check if the `regime` is strictly either `diffusive` or `ballistic`. Right now a simple misspelling like `difusive` would actually generate the ballistic phantom.

Reply:
Good point. I use predefined string variables now instead. Robust enough?